ci-operator/step-registry/crio-test/agent/setup/crio-test-agent-setup.metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+	"path": "assisted/crio-test/setup/assisted-crio-test-setup-ref.yaml",
```

Comment:
this is weird, I'm pretty surprised CI didn't yell about this path being unsynced with the file location